lib/sedna/algorithms/aggregation/aggregation.py

```diff
@@ -121,7 +121,7 @@ def __init__(self, cut_layer, epsilon=100):
 
         if isinstance(self.parameters["epsilon"], str):
```

Comment:
more simple use `if isinstance(self.parameters["epsilon"], str) and self.parameters["epsilon"].isdigit():`